arlo-client/src/components/Audit/index.tsx

```diff
@@ -93,6 +98,7 @@ const Audit: React.FC<{}> = () => {
       {isAuthenticated &&
       (viewMatch === 'setup' || viewMatch === 'progress') ? (
         <>
+          <StatusBox electionId={electionId} refreshId={refreshId} />
```

Comment:
what does `refreshId` do?

Reply:
@jonahkagan It forces the component to reload and thus refetch the data whenever the `refresh` function is called, which happens whenever one of the views successfully updates the server. That way the status box actually consistently reflects the current status.